README.md

```diff
@@ -143,34 +123,43 @@ This workflow also allows you to modify or customize the HarbourBridge codebase.
 To use the tool on a PostgreSQL database called mydb, run
 
 ```sh
-pg_dump mydb | harbourbridge -driver=pg_dump
+pg_dump mydb > mydb.pg_dump
+harbourbridge schema -source=postgresql < mydb.pg_dump
 ```
 
 To use the tool on a MySQL database called mydb, run
 
 ```sh
-mysqldump mydb | harbourbridge -driver=mysqldump
+mysqldump mydb > mydb.mysqldump
+harbourbridge schema -source=mysql < mydb.mysqldump
 ```
 
 To use the tool on a DynamoDB database, run
 
 ```sh
-harbourbridge -driver=dynamodb
+harbourbridge schema -source=dynamodb
 ```
-More details on running harbourbridge can be found in [Example usage](#example-usage) section.
+
+Note: HarbourBridge accepts pg_dump/mysqldump's standard plain-text format,
+but not archive or custom formats. More details on HarbourBridge example usage
+can be found here:
+
+- [PostgreSQL example usage](sources/postgres/README.md#example-postgresql-usage)
+- [MySQL example usage](sources/mysql/README.md#example-mysql-usage)
+- [DynamoDB example usage](sources/dynamodb/README.md#example-dynamodb-usage)
 
 This command will use the cloud project specified by the `GCLOUD_PROJECT`
 environment variable, automatically determine the Cloud Spanner instance
-associated with this project, convert the source schema to a Spanner schema 
+associated with this project, convert the source schema to a Spanner schema
 (For MySQL/Postgres) or infer a schema from the DynamoDB instance, create a
 new Cloud Spanner database with this schema, and finally, populate this new
-database with the data from the source database. 
-If the project has multiple instances, then list of available instances 
-will be shown and you will have to pick one of the available instances and 
-set the `--instance` flag. The new Cloud Spanner database will have a name of 
-the form `{DRIVER}_{DATE}_{RANDOM}`, where`{DRIVER}` is the drivername 
-specified,`{DATE}`is today's date, and`{RANDOM}` is a random suffix for 
-uniqueness.
+database with the data from the source database.
+If the project has multiple instances, then list of available instances
+will be shown and you will have to pick one of the available instances and
+set the `--instance` flag in target-profile. The new Cloud Spanner database
+will have a name of the form `{SOURCE}_{DATE}_{RANDOM}`, where`{SOURCE}` is
+the source specified,`{DATE}`is today's date, and`{RANDOM}` is a random
```

Comment:
Maybe "is the value of the source flag" instead of "the source specified"?

Reply:
Done.